src/v/ssx/BUILD

```diff
@@ -0,0 +1,124 @@
+load("//bazel:build.bzl", "redpanda_cc_library")
+load("//bazel:test.bzl", "redpanda_cc_btest")
+
+redpanda_cc_library(
+    name = "thread_worker",
+    hdrs = [
+        "include/ssx/thread_worker.h",
+    ],
+    strip_include_prefix = "include",
+    visibility = ["//visibility:public"],
+    deps = [
+    ],
+)
+
+redpanda_cc_library(
+    name = "sformat",
+    hdrs = [
+        "include/ssx/sformat.h",
+    ],
+    strip_include_prefix = "include",
+    visibility = ["//visibility:public"],
+    deps = [
+    ],
+)
+
+redpanda_cc_library(
+    name = "future-util",
```

Comment:
I know the source file is an odd one in our tree using hyphen instead of underscore, but generally bazel style is underscores for delimiters - can we stick with that here?

Reply:
Yes, I began to notice this pattern too. Will change.

Reply:
Fixed